# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: d10ae1bb47a8
# Full Hash: d10ae1bb47a8b1cc81718ace05688e97d583e27f
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-10 21:44:02
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 5: Exempt third-party cookie blocking if the channel is on the exception list. r=cookie-reviewers,bvandersloot,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223384
# ==============================================================================

diff -r 336214ad7891 -r d10ae1bb47a8 dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Fri Jan 10 08:24:29 2025 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Fri Jan 10 08:24:30 2025 +0000
@@ -36,6 +36,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    bool matchName,
                    nsString name,
                    nsCString path,
@@ -46,6 +47,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    nsString name,
                    nsString value,
                    bool session,
@@ -60,6 +62,7 @@
                       bool thirdPartyContext,
                       bool partitionForeign,
                       bool usingStorageAccess,
+                      bool isOn3PCBExceptionList,
                       nsString name,
                       nsString path,
                       bool partitioned,